api/hasura/actions/updateUser.ts

```diff
@@ -25,7 +24,27 @@ async function handler(req: VercelRequest, res: VercelResponse) {
   const { circle_id } = payload;
   const { hasuraAddress: address } = session_variables;
 
-  const user = await queries.getUserAndCurrentEpoch(address, circle_id);
+  const {
+    users: [user],
+  } = await adminClient.query({
+    users: [
+      {
+        limit: 1,
+        where: {
+          address: { _ilike: address },
+          circle_id: { _eq: circle_id },
+          // ignore soft_deleted users
+          deleted_at: { _is_null: true },
+        },
+      },
+      {
+        id: true,
+        fixed_non_receiver: true,
+        give_token_received: true,
+      },
+    ],
+  });
+
```

Comment:
why write out the query without deleting the query from the library?

Reply:
`getUserAndCurrentEpoch` is used in other places. Just was overkill here.